doc/application_workflow/README.md

```diff
@@ -0,0 +1,56 @@
+# Container Workflow
+
+The following are guideline suggestions for those who want to use VIC to develop and deploy a containerized application.  These guidelines pertain to VIC 0.7.0.  While VIC continues to progress towards VIC 1.0, the current feature set requires some care in creating a containerized application and deploying it.  We present these guidelines from a developer and a devops perspective.
+
+An example workflow is presented here, in the form of a modified voting app, based on Docker's app.
+
+## Container Workflow Feature Set
+
+General feature set (e.g. Docker run, ps, inspect, etc) used at the CLI will not be discussed.  Only feature set important for containerizing apps and deploying them are discussed.  These include volume and network support.  It is also worth mentioning that basic Docker Compose support is available for application deployment.
+
+#### Currently Available Features
+
+1. Docker Compose (basic)
+2. Registry pull from docker hub and private registry
+3. Named Data Volumes
+4. Anonymous Data Volumes
+5. Bridged Networks
+6. External networks
+7. Port mapping
+8. Network links/alias
+
+#### Future Features
+
+Be aware the following feature must be taken into account when containerizing an app and deploying it.
```

Comment:
`Be aware that the following features are not yet available which must be taken into account when containerizing an app and deploying it`